app/src/org/commcare/activities/LoginActivity.java

```diff
@@ -415,7 +415,6 @@ public void raiseMessage(NotificationMessage message, boolean showTop) {
                     new String[]{toastText});
         }
         uiController.setErrorMessageUI(toastText);
-        Toast.makeText(this, toastText, Toast.LENGTH_LONG).show();

```

Comment:
Seems totally unnecessary and distracting to have a toast pop up to show the exact same message that is sitting on the screen in red/bold text.